atomic_cache.gemspec

```diff
@@ -13,23 +13,28 @@ Gem::Specification.new do |spec|
   spec.summary       = 'summary'
   spec.description   = 'desc'
 
-  spec.licenses      = ['apache 2.0']
+  spec.licenses      = ['Apache-2.0']
   spec.homepage      = 'https://github.com/ibotta/atomic_cache'
 
-  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
-    f.match(%r{^(test|spec|features)/})
-  end
+  spec.files             = %w( README.md LICENSE )
+  spec.files            += Dir.glob("lib/**/*")
+  spec.files            += Dir.glob("docs/**/*")
+  spec.files            += Dir.glob("spec/**/*")
 
   spec.require_paths = ['lib']
 
   # Dev dependencies
   spec.add_development_dependency 'bundler', '~> 1.14'
+  spec.add_development_dependency 'gems', '~> 1.0'
+  spec.add_development_dependency 'git', '~> 1.3'
+  spec.add_development_dependency 'github_changelog_generator', '>= 1.15.0.pre.rc'
+  spec.add_development_dependency 'octokit', '~> 4.0'
```

Comment:
Do we want to trust the `~>` syntax for a public project? `~> 4.0` means `>= 4.0 && < 5.0`. What if one of our dependencies doesn't respect semver and pushes a breaking change on a minor version?

I think I'd advocate for using `=>` for everything just to be safe.

Reply:
dev dependencies are way less breaking, ~> is perfectly fine

Reply:
Hey @trose , good point, but I tend to agree with @onyxraven 's comment for development dependencies.